test/e2e/fixture-builder.js

```diff
@@ -2,14 +2,18 @@ const {
   WALLET_SNAP_PERMISSION_KEY,
   SnapCaveatType,
 } = require('@metamask/snaps-utils');
-const { merge } = require('lodash');
+const { merge, mergeWith } = require('lodash');
 const { toHex } = require('@metamask/controller-utils');
 const { NetworkStatus } = require('@metamask/network-controller');
 
 const { CHAIN_IDS, NETWORK_TYPES } = require('../../shared/constants/network');
 const { SMART_CONTRACTS } = require('./seeder/smart-contracts');
-const { DAPP_URL, DAPP_ONE_URL, ACCOUNT_1 } = require('./helpers');
```

Comment:
instead of using the variables from the helpers file, we use the from the constants file.
This was needed in this PR since otherwise we get an error for test environments not being defined (which is expected as we don't run any test environment with the wallet state, but we do need the fixture-builder methods)